src/storekit.cs

```diff
@@ -797,9 +834,16 @@ interface SKProductStorePromotionController {
 	[DisableDefaultCtor] // Not specified but very likely
 	interface SKStoreReviewController {
 
+		[Deprecated (PlatformName.iOS, 9, 0, message : "Use the 'RequestReview (UIWindowScene windowScene)' API instead.")]
```

Comment:
nope, because
* this API was added in iOS 10.3 - so it can't be deprecated in iOS 9.0
* `UIWindowScene` was added in iOS 13 (iirc)